src/components/Toolbar.js

```diff
@@ -247,13 +248,13 @@ const Toolbar = ({ cursorOnTableView, cursorOnAlphabeticalSort, cursorPinOpen, c
                     : id === 'undo' ? fg
                     : id === 'redo' ? fg
                     : 'gray',
-                    width: fontSize,
-                    height: fontSize
+                    width: fontSize + 4,
+                    height: fontSize + 4,
```

Comment:
@anmolarora1 Why are you adding 4 to the `fontSize` ?

Reply:
The output size of text is slightly different than the output size of the svg icons. We need to add a constant amount to the font size so that they are more evenly matched.